### ISSUE TITLE
Pip Installation not working, error No matching distribution found for mdf-iter

### ISSUE THREAD

**Issue by durgendra** [CSS-Electronics/mdf4-converters#11]
ERROR: Could not find a version that satisfies the requirement mdf-iter (from canedge-browser) (from versions: none)
ERROR: No matching distribution found for mdf-iter

Found error with python version 3.9.13

**Comment by MatinF** [comment]
What OS is this on - and can you share a screenshot of the full console output?

**Comment by durgendra** [comment]
macOS Monterey (12.4). 
I was not getting this error when I used it 2-3 months ago on the same OS but this time I got the error. 

I was doing installation for canedge-influxdb-writer repo but I can't go past the pip install -r requirements.txt because this repo "mdd-iter" is not getting installed. I earlier used on python 3.11 but then downgraded to 3.9 but error remained the same. 

I don't have anything more than logs as below: 

Collecting aiobotocore==2.5.0
  Using cached aiobotocore-2.5.0-py3-none-any.whl (72 kB)
Collecting aiohttp==3.8.1
  Downloading aiohttp-3.8.1-cp39-cp39-macosx_10_9_x86_64.whl (574 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 575.0/575.0 KB 3.8 MB/s eta 0:00:00
Collecting aioitertools==0.10.0
  Using cached aioitertools-0.10.0-py3-none-any.whl (23 kB)
Collecting aiosignal==1.2.0
  Using cached aiosignal-1.2.0-py3-none-any.whl (8.2 kB)
Collecting async-timeout==4.0.2
  Using cached async_timeout-4.0.2-py3-none-any.whl (5.8 kB)
Collecting attrs==21.4.0
  Using cached attrs-21.4.0-py2.py3-none-any.whl (60 kB)
Collecting botocore==1.29.76
  Using cached botocore-1.29.76-py3-none-any.whl (10.4 MB)
Collecting can_decoder>=0.1.9
  Using cached can_decoder-0.1.9-py3-none-any.whl (26 kB)
Collecting canedge-browser>=0.0.8
  Using cached canedge_browser-0.0.8-py3-none-any.whl (17 kB)
Collecting canmatrix==0.9.5
  Using cached canmatrix-0.9.5-py2.py3-none-any.whl (320 kB)
Collecting certifi==2021.10.8
  Using cached certifi-2021.10.8-py2.py3-none-any.whl (149 kB)
Collecting charset-normalizer==2.0.12
  Using cached charset_normalizer-2.0.12-py3-none-any.whl (39 kB)
Collecting click==8.1.2
  Using cached click-8.1.2-py3-none-any.whl (96 kB)
Collecting colorama==0.4.4
  Using cached colorama-0.4.4-py2.py3-none-any.whl (16 kB)
Collecting frozenlist==1.3.0
  Downloading frozenlist-1.3.0-cp39-cp39-macosx_10_9_x86_64.whl (36 kB)
Collecting fsspec==2023.4.0
  Using cached fsspec-2023.4.0-py3-none-any.whl (153 kB)
Collecting future==0.18.3
  Using cached future-0.18.3.tar.gz (840 kB)
  Preparing metadata (setup.py) ... done
Collecting idna==3.4
  Using cached idna-3.4-py3-none-any.whl (61 kB)
Collecting influxdb-client==1.35.0
  Using cached influxdb_client-1.35.0-py3-none-any.whl (719 kB)
Collecting J1939-PGN==0.4
  Using cached J1939_PGN-0.4-py3-none-any.whl (8.8 kB)
Collecting jmespath==0.10.0
  Using cached jmespath-0.10.0-py2.py3-none-any.whl (24 kB)
Collecting multidict==6.0.2
  Downloading multidict-6.0.2-cp39-cp39-macosx_10_9_x86_64.whl (28 kB)
Collecting numpy==1.24.1
  Downloading numpy-1.24.1-cp39-cp39-macosx_10_9_x86_64.whl (19.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 19.8/19.8 MB 26.7 MB/s eta 0:00:00
Collecting pandas==1.5.3
  Using cached pandas-1.5.3-cp39-cp39-macosx_10_9_x86_64.whl (12.0 MB)
Collecting python-dateutil==2.8.2
  Using cached python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
Collecting pytz==2022.1
  Using cached pytz-2022.1-py2.py3-none-any.whl (503 kB)
Collecting reactivex==4.0.4
  Using cached reactivex-4.0.4-py3-none-any.whl (217 kB)
Collecting s3fs==2023.4.0
  Using cached s3fs-2023.4.0-py3-none-any.whl (28 kB)
Collecting six==1.16.0
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting typing_extensions==4.1.1
  Using cached typing_extensions-4.1.1-py3-none-any.whl (26 kB)
Collecting urllib3==1.26.9
  Using cached urllib3-1.26.9-py2.py3-none-any.whl (138 kB)
Collecting wrapt==1.14.0
  Downloading wrapt-1.14.0-cp39-cp39-macosx_10_9_x86_64.whl (35 kB)
Collecting yarl==1.7.2
  Downloading yarl-1.7.2-cp39-cp39-macosx_10_9_x86_64.whl (121 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 121.8/121.8 KB 4.7 MB/s eta 0:00:00
Requirement already satisfied: setuptools>=21.0.0 in ./venv-p3.9/lib/python3.9/site-packages (from influxdb-client==1.35.0->-r requirements.txt (line 19)) (58.1.0)
INFO: pip is looking at multiple versions of can-decoder to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of yarl to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of wrapt to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of urllib3 to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of typing-extensions to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of six to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of s3fs to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of reactivex to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of pytz to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of python-dateutil to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of pandas to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of numpy to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of multidict to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of jmespath to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of j1939-pgn to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of influxdb-client to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of idna to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of future to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of fsspec to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of frozenlist to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of colorama to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of click to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of charset-normalizer to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of certifi to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of canmatrix to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of botocore to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of attrs to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of async-timeout to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of aiosignal to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of aioitertools to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of aiohttp to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of <Python from Requires-Python> to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of aiobotocore to determine which version is compatible with other requirements. This could take a while.
ERROR: Could not find a version that satisfies the requirement mdf-iter (from canedge-browser) (from versions: none)
ERROR: No matching distribution found for mdf-iter
